ckb-bin/src/subcommand/run.rs

```diff
@@ -43,6 +43,8 @@ pub fn run(args: RunArgs, version: Version, async_handle: Handle) -> Result<(),
 
     launcher.check_assume_valid_target(&shared);
 
+    check_reuse_port(launcher.args.config.network.reuse)?;
```

Comment:
Add two methods in `config.network`:

* `is_reuse_supported`: true for linux, false otherwise
* `reuse_enabled()`: return `reuse_hint && is_reuse_supported()`